src/components/core/geoChart/tests/GeoChartRenderer.spec.tsx

```diff
@@ -26,4 +26,20 @@ describe("GeoChartRenderer", () => {
         });
         expect(wrapper.hasClass("s-isExportMode")).toBe(true);
     });
+
+    it.each([[true, true], [false, false]])(
+        "should render GeoChartRenderer component with interactive control is %s",
+        (isDisabledInteractive: boolean, expected: boolean) => {
+            const wrapper = createComponent({
+                config: {
+                    isExportMode: true,
+                    mapboxToken: "",
+                    isDisabledInteractive,
+                },
+            });
+
+            const geoChartRenderProps = wrapper.instance().props as IGeoChartRendererProps;
+            expect(geoChartRenderProps.config.isDisabledInteractive).toBe(expected);
```

Comment:
this test is meaningless.
You passed in a config, then checking for passed in value